### PR TITLE
improved version of client auth

### PR DESCRIPTION
**Pull request by wgerlach**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,13 @@
 
-# docker build -t mgrast/auth-server .
+# docker build -t mgrast/authserver .
 
-# docker rm auth-server ; docker run -d --name auth-server -p 7000:80  mgrast/auth-server
+# docker rm authserver ; docker run -d --name authserver -p 7000:80  mgrast/authserver
```
Comment (wilke) [inline Dockerfile:4]: Did you check if the name change has no side effects?

File: Dockerfile [hunk 2]
```diff
@@ -40,11 +40,14 @@ RUN apt-get install -y \
     
 ENV PERL_MM_USE_DEFAULT 1
 
-RUN mkdir -p /db && chmod a+w /db
-COPY user.db /db/user.db
-RUN chmod a+w /db/user.db
-
-COPY ./cgi-bin ./html /usr/local/apache2/htdocs/
+#RUN mkdir -p /db && chmod a+w /db
```
Comment (wilke) [inline Dockerfile:43]: Why are you taking this out?

File: client/authclient.cgi [hunk 3]
```diff
@@ -57,10 +65,10 @@ my $response = $json->decode($ua->get($call_url)->content);
 my $access_token = $response->{token};
 $call_url = $data_url . "&access_token=" . $access_token;
 $response = $ua->get($call_url)->content;
-my $cookie = CGI::Cookie->new( -name    => COOKIE_NAME,
+my $cookie = CGI::Cookie->new( -name    => AuthConfig::COOKIE_NAME,
```
Comment (wilke) [inline client/authclient.cgi:68]: Why are AuthConfig:* not part of settings?

**Review by wilke — COMMENTED** [review]
How and where tested?

File: Dockerfile (hunk 2 above)
Comment (wgerlach) [reply Dockerfile:43]: I think this is not needed.

File: Dockerfile (hunk 1 above)
Comment (wgerlach) [reply Dockerfile:4]: Yes, I checked.

File: client/authclient.cgi (hunk 3 above)
Comment (wgerlach) [reply client/authclient.cgi:68]: I do not know.

**Pull request merged**